src/main/type.ts

```diff
@@ -0,0 +1,299 @@
+import type { BinaryLike } from 'crypto'
+import type { IncomingMessage } from 'http'
+import type { Readable as ReadableStream } from 'stream'
```

Comment:
shouldnt we use `... from "node:xxx"` for consistency?

Reply:
I personally prefer the node protocol, but I'm not sure if it should be enforced